rfcs/0014-task-workers.md

```diff
@@ -161,10 +162,24 @@ metadata:
     name: pi
 task:
     path: lib/tasks/pi.js
+    # Option 1
     processors:
     	computePi: lib/tasks/piProcessor.js
+
+ 	# Option 2
+    processors:
+    	- name: computePi
+    	  path: lib/tasks/piProcessor.js
+
+
+ 	# Option 3
+    processors:
+    	computePi:
+    	  path: lib/tasks/piProcessor.js
 ```
 
+**TODO:** Decide on configuration style. Option 1 does not allow for introducing additional per-processor configuration in the future (e.g. max CPU threads, priority, etc.)
```

Comment:
I find option 3 the most consistent one, as it has a very similar structure as the task itself, and order does not matter (right?), so there's no reason for option 2.

Reply:
I agree and updated that part